### DISCUSSION TITLE
Ability to Save Search Filters

### DISCUSSION THREAD

**Discussion by gmartelly-suki** [nextgenhealthcare/connect#5426] [Feature/Enhancement Ideas]
Doing an advanced search for the same type of filter over and over is tedious. I'd like the ability to save often used advanced search filters and load them with a click from a button list.

It could be a drop down arrow button attached to the "Advanced..." Search Filter button. It should allow you to choose a name for the filter and also edit it as needed. 

<img width="507" alt="image" src="https://user-images.githubusercontent.com/109001389/190927595-2500a7f9-dd95-4a86-9ab2-b605aed8e235.png">

These custom filters could be available on a channel level and be able to be exported with the channel, or at the global level and saved to the database, or both.

**Comment by pacmano1** [comment]
You should add your request to the issues/feature section here on Mirth Github.  But make sure it has not been submitted before, it does look familiar.

**Reply by jonbartels** [reply]
https://github.com/nextgenhealthcare/connect/search?q=save+search&type=issues

I think the following issues are related:
* https://github.com/nextgenhealthcare/connect/issues/2667
* https://github.com/nextgenhealthcare/connect/issues/3615
* https://github.com/nextgenhealthcare/connect/issues/4322

I think @gmartelly-suki should comment on https://github.com/nextgenhealthcare/connect/issues/3615 . Please say something like "this affects me too. My use case is....". If you have any ideas about how the UI might look, please share those. Plain language use cases helps the product owner understand the user benefits. Give a "thumbs up" react on #3615, upvoted issues get more attention from the product owner. This takes work but the people who make decisions about MC do not use the tool day to day, explaining the benefit and idea in detail makes it easier for them to say yes.

@pacmano1 or @tonygermano can reopen https://github.com/nextgenhealthcare/connect/issues/3615 .
